app/code/Magento/Catalog/Model/Layer/FilterList.php

```diff
@@ -44,18 +47,27 @@ class FilterList
      */
     protected $filters = [];
 
+    /**
+     * @var LayerCategoryConfig|null
+     */
+    private $layerCategoryConfig;
+
     /**
      * @param \Magento\Framework\ObjectManagerInterface $objectManager
      * @param FilterableAttributeListInterface $filterableAttributes
      * @param array $filters
+     * @param LayerCategoryConfig|null $layerCategoryConfig
      */
     public function __construct(
         \Magento\Framework\ObjectManagerInterface $objectManager,
         FilterableAttributeListInterface $filterableAttributes,
-        array $filters = []
+        array $filters = [],
+        LayerCategoryConfig $layerCategoryConfig = null
     ) {
         $this->objectManager = $objectManager;
         $this->filterableAttributes = $filterableAttributes;
+        $this->layerCategoryConfig = $layerCategoryConfig ?:
```

Comment:
Since we are bringing changes to the minor release, it's not necessary to invoke the object manager here. You may remove it.